extract-metadata.py

```diff
@@ -44,7 +70,10 @@ def processingInstruction(self, target, data):
         fileext = os.path.splitext(filename)[1]
         extract_func = extract_funcs.get(fileext, lambda a: {})
         frontmatter = extract_func(filename)
-        value = frontmatter.get(target, "")
+        if target == "title" and frontmatter.get("abbrev", None) != None:
```

Comment:
OK, github won't let me do a multi-line suggestion here for some reason, so you get two...

```suggestion
        if target == "abbrev":
```